app/(tabs)/explore.tsx

```diff
@@ -1,29 +1,61 @@
-import Ionicons from '@expo/vector-icons/Ionicons';
-import { StyleSheet, Image, Platform } from 'react-native';
+import Ionicons from "@expo/vector-icons/Ionicons";
+import { useEffect, useState } from "react";
+import { StyleSheet, Image, Text, View } from "react-native";
 
-import { Collapsible } from '~/components/Collapsible';
-import { ExternalLink } from '~/components/ExternalLink';
-import ParallaxScrollView from '~/components/ParallaxScrollView';
-import { ThemedText } from '~/components/ThemedText';
-import { ThemedView } from '~/components/ThemedView';
+import { Collapsible } from "~/components/Collapsible";
+import { ExternalLink } from "~/components/ExternalLink";
+import ParallaxScrollView from "~/components/ParallaxScrollView";
+import { ThemedText } from "~/components/ThemedText";
+import { ThemedView } from "~/components/ThemedView";
+
+import { collection, getDocs } from "firebase/firestore";
+import { db } from "~/firebase/config";
 
 export default function TabTwoScreen() {
+  const [testData, setTestData] = useState<any>([]);
```

Comment:
Nit - would prefer these typed but we can fix later

Reply:
yeah, without adding some other parts of the repo we can't really do that.